modules/questions.py

```diff
@@ -128,14 +128,39 @@ def __init__(self) -> None:
         self.last_posted_time: datetime = (
             datetime.now() - self.AUTOPOST_QUESTION_INTERVAL / 2
         )
-        # Was the last question that was posted, automatically posted by Stampy?
-        self.last_question_autoposted = False
+        # regexes
+        self.re_post_question = re.compile(
+            r"""
+            (?:get|post|next) # get / post / next
+            \s # whitespace char (obligatory)
+            (?:\d+\s)? # optional number of questions
+            (?:q|questions?|a|answers?|it|last)
+            # q / question / questions / a / answer / answers / it / last
+            """,
+            re.I | re.X,
+        )
+        self.re_get_question_info = re.compile(r"(?:i|info|get info)\b", re.I)
+        self.re_count_questions = re.compile(
+            r"(?:count|how many|number of|n of|#) (?:q|questions|a|answers)", re.I
+        )
+        self.re_big_next_question = re.compile(  # TODO: this matches just "a question"
+            r"(([wW]hat(’|'| i)?s|([Cc]an|[Mm]ay) (we|[iI]) (have|get)|[Ll]et[’']?s have|[gG]ive us)"
+            r"?( ?[Aa](nother)?|( the)? ?[nN]ext) question,?( please)?\??|([Dd]o you have|([Hh]ave you )"
+            r"?[gG]ot)?( ?[Aa]ny( more| other)?| another) questions?( for us)?\??)!?"
+        )
+        self.re_big_count_questions = re.compile(
+            r"([hH]ow many questions (are (there )?)?(left )?in)|([hH]ow "
+            r"(long is|long's)) (the|your)( question)? queue( now)?\??",
+        )
         self.re_refresh_questions = re.compile(
             r"(reload|fetch|load|update|refresh)(\s+new)?\s+q(uestions)?", re.I
         )
 
+        # Was the last question that was posted, automatically posted by Stampy?
+        self.last_question_autoposted = False
+
         # Register `post_random_oldest_question` to be triggered every after 6 hours of no question posting
-        @self.utils.client.event
+        @self.utils.client.event  # TODO this should be registered on Rob's discord only
```

Comment:
done

Reply:
sorry, I wanted to ask to UPDATE the comment with an issue number, not DELETE the comment completely 😓 but the issue points to the "right" place in code that needs updating, so not necessary to have bi-directional links I guess...

just for future, it would be ideal if every TODO comment that makes it into a PR pointed to some issue number and wasn't just handing in the code without any tracking number whatsoever 😅